reviewing-prs.zh.md

```diff
@@ -1,101 +1,101 @@
 ---
-title:  "如何 Review 一个 Pull Request"
+title:  "如何审核一个 Pull Request"
 ---
 
 <hr />
 
-This guide is for all committers and contributors that want to help with reviewing code contributions. Thank you for your effort - good reviews are one the most important and crucial parts of an open source project. This guide should help the community to make reviews such that:
+本指南适用于希望帮助审核代码的所有提交者和贡献者。感谢你的努力 - 良好的审核是开源项目中最重要也是最关键的部分。本指南应协助社区进行审核，包括以下几个方面:
 
-* Contributors have a good contribution experience.
-* Our reviews are structured and check all important aspects of a contribution.
-* We make sure to keep a high code quality in Flink.
-* We avoid situations where contributors and reviewers spend a lot of time refining a contribution that gets rejected later.
+* 贡献者有很好的贡献经验。
+* 我们的审核是有条理的，并且检查贡献的所有重要方面。
+* 我们确保在 Flink 中保持高质量的代码。
+* 我们避免了贡献者和审核者花费大量时间来完善后来会被拒绝的贡献的情况。
 
 ----
 
 {% toc %}
 
-## Review Checklist
+## 审核清单
 
-Every review needs to check the following five aspects. We encourage to check these aspects in order, to avoid spending time on detailed code quality reviews when there is no consensus yet whether a feature or change should actually be added.
+每次审核都需要检查以下五个方面，我们鼓励大家按顺序检查以下这些方面，当是否应实际添加或更改功能尚未达成共识时，避免在代码质量的审核上花费时间。
```

Comment:
We encourage to check these aspects in order, to avoid spending time on detailed code quality reviews when there is no consensus yet whether a feature or change should actually be added.

我们建议按照以下顺序进行检查，以避免在还没有就是否添加某项功能或需要改动达成共识之前，就花费时间进行详细的代码质量审核。